WordPress/Classes/ViewRelated/Aztec/Stock Photos/StockPhotosPlaceholder.swift

```diff
@@ -24,6 +27,32 @@ final class StockPhotosPlaceholder: WPNoResultsView {
     }
 
     private func configureSubtitle() {
-        messageText = .freePhotosPlaceholderSubtitle
+        do {
+            attributedMessageText = try createStringWithLinkAttributes(from: .freePhotosPlaceholderSubtitle)
+        } catch {
+            // A translation error could make the creation of link attributes to fail. (i.e. removing the '{')
+            // This will make sure that the message is still present, without any {}, but without the link.
+            messageText = removeCurlybraces(from: .freePhotosPlaceholderSubtitle)
+        }
+    }
+
+    private func createStringWithLinkAttributes(from subtitle: String) throws -> NSAttributedString {
```

Comment:
Fwiw we've done something similar for the terms of service link during signup: https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Extensions/String+underline.swift#L9

However, in this case, I doubt `Pexels` would ever be translated to something different since it's a company name. What about instead of wrapping it in braces, you replace all occurances of `Pexels` with a link?

Reply:
Yeah, that's right! I thought maybe in a extreme case it might change but as a company name It's not supposed to. I can add that to the translators notes.

I will also check for those already existing ways to manage the attributes. Thank you!